misc/labels.md

```diff
@@ -1,39 +1,39 @@
-# How we use github labels
+# How we use Github labels
 
-*Because only team-mermbers can add and change labels this document is mainly for maintainers, but also for users to understand how we use labels.*
+*Because only team members can add and change labels this document is mainly for maintainers, but also for users to understand how we use labels.*
 
-*It is important to also label old and closed issues uniformly to be able to export them later r.g. if the project gets separated into multiple components.*
+*It is important to also label old and closed issues uniformly in order to export them later e.g. if the project gets separated into multiple components.*
 
 
 ## Issue Types
-If an issue was created it always MUST be labeled as one of the following issue types:
+If an issue was created it MUST always be labeled as one of the following issue types:
 
 ### `Question`
 The author has a general or very specific question.<br>
-If it is a general question how to use vis.js the issues should be closed immediately with a reference to [stackoverflow](https://stackoverflow.com/questions/tagged/vis.js).<br>
+If it is a general question on how to use vis.js the issues should be closed immediately with a reference to [stackoverflow](https://stackoverflow.com/questions/tagged/vis.js).<br>
 Specific question or hard problems should stay open.<br>
-Questions should be closes within 3 month.
+Questions should be closed within 3 months.
 
 ### `Problem`
 This issues points to a potential bug that needs to be confirmed.<br>
-If the problem is probably in the users code it should be better labeled as `Question`.<br>
-The support-Team should try to reproduce this issue and than close it or mark it as `Confirmed Bug`:<br>
+If the problem most likely originates from the user's code it should be better labeled as `Question`.<br>
+The support team should try to reproduce this issue and then close or mark it as `Confirmed Bug`:<br>
 `Problem` -> `Confirmed Bug` --> `Work In Progress` --> `Fixed awaiting release`
```

Comment:
I really like these label 'graphs'; they clarify the workflow.

Only, I would prefer to have these bundled together in their own section, also because they mention several labels, not just one. Perhaps a section at the top of the document with title 'Overview' or at the bottom with title 'Workflow'. 

Ideally, the labels would be clickable, so that you can just to the label description.